### DISCUSSION TITLE
forms: ability to disable rings and outlines

### DISCUSSION THREAD

**Discussion by TheDevMinerTV** [tailwindlabs/tailwindcss#12143] [Ideas]
From what I've understood @tailwindcss/forms is a **basic** reset for the stupid default styling of form related HTML elements.
> A plugin that provides a basic reset for form styles that makes form elements easy to override with utilities. - from the repo's description

It **shouldn't** put a ring and border onto everything that it touches without it at least being configurable (which it isn't).

(I was directed here by the @tailwindcss/forms repo's "New issue" button)

**Comment by TheDevMinerTV** [comment]
I've gone ahead and created a PR for the forms plugin here that allows disabling the rings and outline: https://github.com/tailwindlabs/tailwindcss-forms/pull/150

**Comment by peacocksalon** [comment]
Thanks for sharing this, this is amazing. i like it.  [salon in navi mumbai](https://www.peacocksalon.in/)
